modules/test/playwright/tests/commerce/commerceCatalog.spec.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {applicationsMenuPageTest} from '../../fixtures/applicationsMenuPageTest';
+import {commercePagesTest} from '../../fixtures/commercePagesTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	applicationsMenuPageTest,
+	commercePagesTest,
+	dataApiHelpersTest,
+	loginTest()
+);
+
+test('LPD-3185 Search a catalog entry using global search, click on a suggested entry and get redirected to that product details page', async ({
+	apiHelpers,
+	applicationsMenuPage,
+	commerceCatalogPage,
+	page,
+}) => {
+	const site = await apiHelpers.headlessSite.createSite({
+		name: 'Minium',
+		templateKey: 'minium-initializer',
+		templateType: 'site-initializer',
+	});
+
+	await apiHelpers.commerceMiniumApiHelper.loadCleanUpData(site);
```

Comment:
Use the utility done here: https://github.com/liferay-commerce/liferay-portal/pull/4994/commits/63db7c5f40383f785c331923be43682541ea2f0a